packages/ui/src/components/ContentReference/GroupReference.tsx

```diff
@@ -24,14 +31,15 @@ export function GroupReference({ groupId }: { groupId: string }) {
   }, [group, onPressGroupRef]);
 
   return (
-    <Reference onPress={onPress}>
+    <Reference onPress={onPress} viewMode={viewMode}>
       <Reference.Header>
         <Reference.Title>
           <ContactAvatar contactId={host} size="$xl" />
           <ContactName
             color="$tertiaryText"
             size="$s"
             userId={host}
+            maxWidth={230}
```

Comment:
Same here

Reply:
And if this is going to be used in a few places, maybe we define it in one place and import it where we need it.